config/src/main/java/com/typesafe/config/impl/Parser.java

```diff
@@ -319,6 +319,16 @@ private TokenWithComments nextTokenIgnoringNewline() {
             return t;
         }
 
+        // Grabs the next Token off of the TokenIterator, ignoring
+        // IgnoredWhitespace tokens
+        private Token nextTokenFromIterator() {
```

Comment:
if this is called nextTokenIgnoringWhitespace or something it might be a little clearer (fromiterator isn't super relevant to the caller)